package.json

```diff
@@ -35,7 +35,8 @@
     "pug": "^2.0.4",
     "tmp": "^0.1.0",
     "winston": "^3.2.1",
-    "yaml": "^1.6.0"
+    "yaml": "^1.6.0",
+    "yarn": "^1.22.11"
```

Comment:
Why is this added?

Reply:
This slipped trough somehow, will remove :)